tests/test_cli_compile.py

```diff
@@ -458,8 +458,32 @@ def test_quiet_option(runner):
     with open("requirements", "w"):
         pass
     out = runner.invoke(cli, ["--quiet", "-n", "requirements"])
-    # Pinned requirements result has not been written to output
-    assert "Dry-run, so nothing updated." == out.stderr.strip()
+    # Pinned requirements result has not been written to output. (An empty
+    # out.stderr results in a ValueError raised by Click)
+    try:
+        assert "" == out.stderr.strip()
+    except ValueError:
+        pass
+
+
+def test_dry_run_noisy_option(runner):
+    with open("requirements", "w"):
+        pass
+    out = runner.invoke(cli, ["--dry-run", "-n", "requirements"])
```

Comment:
Flags `--dry-run` and `-n` are the same, see `pip-compile --help`:
```
  -n, --dry-run                   Only show what would happen, don't change
```

Have you intentionally use these flags both?

Reply:
This was an oversight on my part. I've removed the redundant `-n`.